packages/client/tests/functional/methods/upsert/native-atomic/tests.ts

```diff
@@ -188,34 +188,15 @@ testMatrix.setupTestSuite(
       expect(checker.usedNative()).toBeTruthy()
     })
 
-    testIf(getQueryEngineProtocol() !== 'json')(
-      'should only use ON CONFLICT when there is only 1 unique field in the where clause',
-      async () => {
-        const name = faker.name.firstName()
-
-        await expect(() =>
-          // This will fail
-          client.user.upsert({
-            where: {
-              // Because two unique fields are used
-              id: '1',
-              name,
-            },
-            create: {
-              name,
-            },
-            update: {
-              name,
-            },
-          }),
-        ).rejects.toThrow('Argument where of type UserWhereUniqueInput needs exactly one argument')
-
-        const checker = new UpsertChecker(client)
+    test('should only use ON CONFLICT when there is only 1 unique field in the where clause', async () => {
```

Comment:
`getQueryEngineProtocol` import can probably be removed unless I've missed it being used elsewhere in the file